app/src/main/java/org/p2p/wallet/striga/user/repository/StrigaUserStatusDestinationMapper.kt

```diff
@@ -17,9 +17,12 @@ class StrigaUserStatusDestinationMapper {
             userStatus?.isMobileVerified == false -> {
```

Comment:
you can create local not nullable variable 
`val status = userStatus ?: return `
and avoid compare nullable value with true or false

Reply:
I'm going to remove isUserNotCreated and use userStatus directly to eliminate nullable comparisons through automatic non-null casting